### ISSUE TITLE
help() make Python3 shell crash

### ISSUE THREAD

**Issue by 2gn** [holzschu/a-shell#495]
# To reproduce 
1. In python's interactive shell, import random library.
2. Execute `help(LIBRARY_NAME)`
3. Quit it
4. Crash

```shell
$ python3
>>> import datetime
>>> help(datetime)
# help for the module "datetime" will be shown
# quit it by hitting "q"
>>> help(datetime) # cannot type anything anymore

``` 

To make it work again, you need to restart a-Shell.

**Comment by holzschu** [comment]
Thanks for raising the issue; I think I see what's happening (and maybe how to fix it).

Note that the problem won't happen with `ipython` (which is much better than `python` for interactive commands).

**Comment by 2gn** [comment]
I see. I will use ipython instead then. Thank you!

**Comment by holzschu** [comment]
Hi, this issue will be fixed in version 1.10.2 (the next one).

**Comment by 2gn** [comment]
> Hi, this issue will be fixed in version 1.10.2 (the next one).

Thank you!
